lib/faktory_worker/push_pipeline/consumer.ex

```diff
@@ -0,0 +1,27 @@
+defmodule FaktoryWorker.PushPipeline.Consumer do
+  @moduledoc false
+
+  alias FaktoryWorker.{ConnectionManager, Pool}
+
+  @behaviour Broadway
+
+  @impl true
+  def handle_message(_processor_name, message, _context), do: message
+
+  @impl true
+  def handle_batch(_, [%{data: job} = message], _batch_info, %{name: name}) do
+    result =
+      name
+      |> Pool.format_pool_name()
+      |> :poolboy.transaction(
+        &ConnectionManager.send_command(&1, {:push, job}),
+        5000
```

Comment:
What is the `5000` used for?

Reply:
This is the timeout to get a connection from the pool

Reply:
Do we want a shorter timeout than that? Wouldn't that mean something was wrong if it was taking that long?